website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -1171,7 +1171,7 @@ Handlers for File Data Sources cannot be in files which import `eth_call` contra
 
 #### Examples
 
-[Crypto Coven Subgraph migration](https://github.com/azf20/cryptocoven-api/tree/file-data-sources-refactor)
+[Crypto Coven Subgraph upgrade](https://github.com/azf20/cryptocoven-api/tree/file-data-sources-refactor)
```

Comment:
This shouldn't change, this "migration" refers to migrating from `ipfs.cat` to File Data Sources

Reply:
Got it. Good catch!